src/hotspot/share/nmt/memBaseline.hpp

```diff
@@ -53,7 +53,7 @@ class MemBaseline {
     by_address,      // by memory address
     by_size,         // by memory size
     by_site,         // by call site where the memory is allocated from
-    by_site_and_type // by call site and memory tag
+    by_site_and_tag // by call site and memory tag
```

Comment:
Move out into mainline PR (and indentation of comment seems wrong)

Reply:
Done.